Pod/INSImageView.swift

```diff
@@ -257,7 +296,7 @@ public class INSImageView: UIImageView {
     
     public override var image: UIImage? {
         get {
-            return nil
+            return imageView.image
```

Comment:
i was using it to get the UIImage it was setted, it could be useful,  so the methods should be redesigned to accept the image not nil returned